lib/app/admin/data-reset/reset-users.js

```diff
@@ -0,0 +1,27 @@
+var Promise = require('bluebird');
+var $fh = require('fh-mbaas-api');
+
+var authServiceGuid = process.env.WFM_AUTH_GUID;
+
+module.exports = function() {
+
+  return Promise.fromCallback(function(fromCallbackCb) {
+    $fh.service({
+      guid: authServiceGuid,
+      path: "/admin/data",
+      method: "DELETE",
+      timeout: 25000
+    }, function(err, body, res) {
+      //resolve manually. $fh.service returns error on internal (fh) or connection error.
+      // we can check statusCode and body for service side errors - code !== 204
+      if (err) {
+        fromCallbackCb(new Error(err));
+      } else if (res && res.statusCode !== 204) {
```

Comment:
A 200 would be also be a valid http response also. 

Could use `else if (res && res.statusCode && res.statusCode > 204) {` to catch error http status codes.